File: C++/Prime.cpp
#include<iostream>
using namespace std;
main(){
	int n;
	cout<<"Enter the value of n :";
	cin>>n;
	bool isPrime=1;
	for(int i=2;i<n;i++)
		if(n%i==0){
			isPrime=0;
			break;
		}
		if(isPrime==0){
		cout<<"The number is not a Prime Number.";
	}
	else{
		cout<<"The number is a Prime Number";
	}
	return 0;
}

File: Java/Prime.java
import java.util.Scanner;

public class Prime {

     static boolean isPrime(int num) {
          if (num <= 1) {
               return false;
          }

          for (int i = 2; i * i <= num; i++) {
               if (num % i == 0) {
                    return false;
               }
          }

          return true;
     }

     public static void main(String[] args) {
          Scanner sc = new Scanner(System.in);
          System.out.print("Enter the number: ");
          int num = sc.nextInt();

          boolean prime = isPrime(num);

          if (prime) {
               System.out.println(num + " is a prime number.");
          } else {
               System.out.println(num + " is not a prime number.");
          }

          sc.close();
     }
}
